src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Cohost/HtmlFormattingCollection.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Razor.Formatting;
+using Xunit;
+
+namespace Microsoft.VisualStudio.Razor.LanguageClient.Cohost;
+
+[CollectionDefinition(Name)]
+public class HtmlFormattingCollection : ICollectionFixture<HtmlFormattingFixture>
```

Comment:
IIRC, the last time I added a collection fixture, I ended up using a shared project to avoid copying and pasting code since xUnit insists that these be defined in the test assembly.